### PR TITLE
Add make commands to Laravel5 and Lumen modules

### PR DESCRIPTION
**Pull request by ibpavlov**
Add `make` and `makeMultiple` commands to [Laravel5] and [Lumen] modules. 
They will act like factory()->make()

### CODE REVIEW

File: src/Codeception/Module/Laravel5.php [hunk 1]
```diff
@@ -1167,6 +1167,63 @@ public function haveMultiple($model, $times, $attributes = [], $name = 'default'
...
+     *
+     * ``` php
+     * <?php
+     * $I->make('App\User');
+     * $I->make('App\User', ['name' => 'John Doe']);
+     * $I->make('App\User', [], 'admin');
+     * ?>
+     * ```
+     *
+     * @see https://lumen.laravel.com/docs/master/testing#model-factories
```
Comment (Naktibalda) [inline src/Codeception/Module/Laravel5.php]: You left Lumen links in Laravel docblocks. Please fix them.
markdown files are regenerated by release process, so the changes you made there will be lost if they aren't the same as the changes in code files.

Comment (ibpavlov) [reply src/Codeception/Module/Laravel5.php]: Ok. Thank I'll fix them. 
For your previous comment: I've added that chamge to the DataFactory, but for Laravel it utilizes their factories.

File: src/Codeception/Module/Laravel5.php [hunk 2]
```diff
@@ -1167,6 +1167,63 @@ public function haveMultiple($model, $times, $attributes = [], $name = 'default'
...
+     *
+     * ``` php
+     * <?php
+     * $I->make('App\User');
+     * $I->make('App\User', ['name' => 'John Doe']);
+     * $I->make('App\User', [], 'admin');
+     * ?>
+     * ```
+     *
+     * @see http://laravel.com/docs/5.1/testing#model-factories
```
Comment (DavertMik) [inline src/Codeception/Module/Laravel5.php]: I would ask you to write a bit more how data factories are defined. Link to laravel website is not enough as it doesn't say how to use it with codeception

**Comment by Naktibalda** [comment]
@ibpavlov It would be great if you could make another pull request with more documentation about these methods.

**Pull request merged**